config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -404,6 +404,66 @@ periodics:
     testgrid-tab-name: aks-engine-azure-windows-master-containerd-serial-slow
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: Runs serial Windows tests on a Kubernetes cluster running containerd provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud
+- interval: 48h
+  name: ci-kubernetes-e2e-aks-engine-master-windows-containerd-azuredisk-csi-driver
+  decorate: true
+  decoration_config:
+    timeout: 2h
+  labels:
+    preset-service-account: "true"
+    preset-azure-cred: "true"
+    preset-azure-windows: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  - org: kubernetes-sigs
+    repo: azuredisk-csi-driver
+    base_ref: master
+    path_alias: sigs.k8s.io/azuredisk-csi-driver
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200623-2424179-master
+      command:
+      - runner.sh
+      - kubetest
+      args:
+      # Generic e2e test args
+      - --test
+      - --up
+      - --down
+      - --build=quick
+      - --dump=$(ARTIFACTS)
+      # Azure-specific test args
+      - --deployment=aksengine
+      - --provider=skeleton
+      - --aksengine-admin-username=azureuser
+      - --aksengine-admin-password=AdminPassw0rd
+      - --aksengine-creds=$(AZURE_CREDENTIALS)
+      - --aksengine-download-url=https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz
```

Comment:
`https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz` is using aks-engine v0.48.0. Do we need to update the version?

Reply:
hmm, let me check, I think we need a version newer than v0.48.0. Maybe I'll just point it to more recent release on github.

Reply:
Updated aks-engine version to v0.53.0.